test/jdk/sanity/client/lib/SwingSet3/src/com/sun/swingset3/demos/table/resources/oscars.xml

```diff
@@ -1 +1,2212 @@
        <lost>
            <movie>General Spanky</movie>
            <org>Hal Roach Studio Sound Department</org>
            <person>Elmer A. Raguse</person></lost></sound>
    <sound year="1936">
        <lost>
            <movie>Mr. Deeds Goes to Town</movie>
            <org>Columbia Studio Sound Department</org>
            <person>John Livadary</person></lost></sound>
    <sound year="1936">
        <lost>
            <movie>The Texas Rangers</movie>
            <org>Paramount Studio Sound Department</org>
            <person>Franklin B. Hansen</person></lost></sound>
    <sound year="1936">
        <lost>
            <movie>That Girl from Paris</movie>
            <org>Paramount Studio Sound Department</org>
            <person>J. O. Aalberg</person></lost></sound>
    <sound year="1936">
        <lost>
            <movie>Three Smart Girls</movie>
            <org>Universal Studio Sound Department</org>
            <person>Homer G. Tasker</person></lost></sound>
    <writing year="1936" subcat="original story">
        <won>
            <movie>The Story of Louis Pasteur</movie>
            <person>Pierre Collings</person>
            <person>Sheridan Gibney</person></won></writing>
    <writing year="1936" subcat="original story">
        <lost>
            <movie>Fury</movie>
            <person>Norman Krasna</person></lost></writing>
    <writing year="1936" subcat="original story">
        <lost>
            <movie>The Great Ziegfeld</movie>
            <person>William Anthony McGuire</person></lost></writing>
    <writing year="1936" subcat="original story">
        <lost>
            <movie>San Francisco</movie>
            <person>Robert Hopkins</person></lost></writing>
    <writing year="1936" subcat="original story">
        <lost>
            <movie>Three Smart Girls</movie>
            <person>Adele Comandini</person></lost></writing>
    <writing year="1936" subcat="screenplay">
        <won>
            <movie>The Story of Louis Pasteur</movie>
            <person>Pierre Collings</person>
            <person>Sheridan Gibney</person></won></writing>
    <writing year="1936" subcat="screenplay">
        <lost>
            <movie>After the Thin Man</movie>
            <person>Frances Goodrich</person>
            <person>Albert Hackett</person></lost></writing>
    <writing year="1936" subcat="screenplay">
        <lost>
            <movie>Dodsworth</movie>
            <person>Sidney Howard</person></lost></writing>
    <writing year="1936" subcat="screenplay">
        <lost>
            <movie>Mr. Deeds Goes to Town</movie>
            <person>Robert Riskin</person></lost></writing>
    <writing year="1936" subcat="screenplay">
        <lost>
            <movie>My Man Godfrey</movie>
            <person>Eric Hatch</person>
            <person>Morris Ryskind</person></lost></writing>
</oscars> 
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
+<oscars xmlns:osc-results="http://www.fatdog.com/oscar-results"

```

Comment:
Does this change affect the behavior of the associated test(s)?

Reply:
I was also wondering how this was tested.

Reply:
FYI, the way to test this is ..
cd test/jdk
jtreg sanity/client/SwingSet/src/TableDemoTest.java

I tried converting all ^M to \n in this file another repo and comparing with your patch but every line was different so I just can't review this.

Reply:
For safety reasons I've restored this file as well as all the files in unit tests.